src/metabase/util/malli/schema.clj

```diff
@@ -333,7 +334,13 @@
      [:slug {:optional true} :string]
      [:name {:optional true} :string]
      [:default {:optional true} :any]
-     [:sectionId {:optional true} NonBlankString]]
+     [:sectionId {:optional true} NonBlankString]
+     [:temporal_units {:optional true}
+      [:sequential
+       (into [:enum]
+             (concat
+              lib.schema.temporal-bucketing/temporal-bucketing-units
+              (map name lib.schema.temporal-bucketing/temporal-bucketing-units)))]]]
```

Comment:
Ehm.. I guess this is not relevant for svg sanitization?

Reply:
oh, oops. Good catch, I'll remove that :)